client/python/README.md

```diff
@@ -9,31 +9,107 @@ pip install djclient
 ## Examples
 
 To initialize the client:
-```
-from djclient import DJClient
+```python
+from djclient.dj import DJClient, Source, Dimension, Transform, Metric
```

Comment:
I see why the `.dj` was added, but is it worth using the `__all__` trick in `__init__.py` for these main classes so we can keep it `from djclient`?

Reply:
Oh definitely -- forgot to add this